docs/how-to-generate-diagrams.md

```diff
@@ -0,0 +1,9 @@
+#How to generate diagrams
+
+Proxy diagrams should be generated by preference with PlantUML.
```

Comment:
Let's provide an embedded link for the first instance of PlantUML:
[PlantUML](http://plantuml.com)